examples/redis/chart.ts

```diff
@@ -0,0 +1,12 @@
+import { Construct } from "constructs";
+import { Redis, TalisChart, TalisChartProps } from "../../lib";
+
+export class RedisChart extends TalisChart {
+  constructor(scope: Construct, props: TalisChartProps) {
+    super(scope, { app: "example-redis-app", ...props });
+
+    new Redis(this, "redis-example", {
+      release: "v1.0",
```

Comment:
I'd like to see a real redis tag here: https://hub.docker.com/_/redis
So that the example can actually be applied.